src/SFA.DAS.LearnerDataMismatches.Web/Pages/Learner.cshtml.cs

```diff
@@ -25,10 +25,16 @@ public class LearnerModel : PageModel
         public string LearnerName { get; set; }
 
         public IEnumerable<string> DataLockNames =>
-            CollectionPeriods
-                .SelectMany(c => c.UniqueDataLockNames)
-                .Distinct()
-                .OrderBy(x => x);
+            NewCollectionPeriods
```

Comment:
This was the bit that was falling over when the ID was wrong in the URL. Can we convert this into a method or populate this later?

Reply:
I have made sure the collections are never `null` and so don't cause `NullReferenceException`